src/creature/main_character/main_character_race_class/main_character_race_class.cpp

```diff
@@ -0,0 +1,187 @@
+#include "main_character_race_class.h"
+
+#include <iostream>
+
+const std::vector<main_character_race::MCRace> main_character_race::AVAILABLE_MCRACES_LIST = {
+	main_character_race::MCRace::DwarfHill,
+	main_character_race::MCRace::DwarfMountain,
+	main_character_race::MCRace::ElfHigh,
+	main_character_race::MCRace::ElfWood,
+	main_character_race::MCRace::Human,
+	main_character_race::MCRace::Tiefling};
+
+std::string main_character_race::MCRaceToString(main_character_race::MCRace player_race)
+{
+	switch (player_race)
+	{
+		case main_character_race::MCRace::DwarfHill:
+			return "Dwarf (Hill)";
+			break;
+		case main_character_race::MCRace::DwarfMountain:
+			return "Dwarf (Mountain)";
+			break;
+		case main_character_race::MCRace::ElfHigh:
+			return "Elf (High)";
+			break;
+		case main_character_race::MCRace::ElfWood:
+			return "Elf (Wood)";
+			break;
+		case main_character_race::MCRace::Human:
+			return "Human";
+			break;
+		case main_character_race::MCRace::Tiefling:
+			return "Tiefling";
+			break;
+		case main_character_race::MCRace::Undefined:
+		default:
+			return "Undefined";
+	}
+}
+
+main_character_race::MCRace main_character_race::StringToMCRace(std::string player_race)
+{
+	if (player_race == "Hill Dwarf")
+	{
+		return main_character_race::MCRace::DwarfHill;
+	}
+	else if (player_race == "Mountain Dwarf")
+	{
+		return main_character_race::MCRace::DwarfMountain;
+	}
+	else if (player_race == "High Elf")
+	{
+		return main_character_race::MCRace::ElfHigh;
+	}
+	else if (player_race == "Wood Elf")
+	{
+		return main_character_race::MCRace::ElfWood;
+	}
+	else if (player_race == "Human")
+	{
+		return main_character_race::MCRace::Human;
+	}
+	else if (player_race == "Tiefling")
+	{
+		return main_character_race::MCRace::Tiefling;
+	}
+	else
+	{
+		std::cerr << "Race \"" << player_race
+				  << "\" not recognized. Setting it to \"Undefined\".\n";
+		return main_character_race::MCRace::Undefined;
+	}
+}
```

Comment:
When i try to select one of the races in the list it is always wrong and set to undefined, for example in the list we present Dwarf (Hill) and expect as a correct answer Hill Dwarf

Reply:
You are right. I am changing that so that the two are aligned. Let's keep the `Dwarf (Hill)` approach (and similarly for the other instances).

Reply:
Actually, that would require the user to type `(` which is currently forbidden by the `GetUserInput` function. Let's stick with the other method then...